index.html

```diff
@@ -129,7 +129,7 @@ <h2>Multi-Post <br>
     </section>
 
     <!-- About myself section -->
-    <!-- <section class="about_myself">
+     <section class="about_myself">
         <h2>About Myself</h2>
             <p>Hello I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
```

Comment:
- [ ] Kindly align this section using **flexbox**. Because it would give us more flexibility to layout our content and it is responsive 😉👌